src/tests/sqlsmith/src/relation.rs

```diff
@@ -70,6 +70,13 @@ impl<'a, R: Rng> SqlGenerator<'a, R> {
         (table_factor, columns)
     }
 
+    fn gen_local_table_factor(&mut self) -> (TableFactor, Vec<Column>) {
```

Comment:
"local_table_factor" looks weird, what "local" indicates?

Reply:
Currently default behaviour is that `gen_table_factor` can directly mutate global bound relations.

`local` indicates this won't happen, changes to context are local.

This should be default behaviour of `gen_table_factor`. Will update accordingly.